Dockerfile

```diff
@@ -37,27 +37,25 @@ RUN set -xe \
 # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
 ENV COMPOSER_ALLOW_SUPERUSER 1
 
-RUN composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --optimize-autoloader --classmap-authoritative \
-	&& composer clear-cache
+# Use prestissimo to speed up builds
+RUN composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --optimize-autoloader --classmap-authoritative
+
+COPY docker/app/docker-entrypoint.sh /usr/local/bin/docker-app-entrypoint
+RUN chmod +x /usr/local/bin/docker-app-entrypoint
+
+# Download the Symfony skeleton and leverage Docker cache layers
+ENV SKELETON_COMPOSER_JSON https://raw.githubusercontent.com/symfony/skeleton/v3.3.4/composer.json
 
 WORKDIR /srv/app
+RUN php -r "copy('$SKELETON_COMPOSER_JSON', 'composer.json');" \
```

Comment:
I think you could use `ADD ` here

Reply:
No because the file is not included in the repo. But I’ll switch to composer create-project in another PR.

Reply:
? `ADD` works with remote urls...

Reply:
I wasn't aware of that! great

Reply:
Yeah I was pretty happy too when I realised I didn't actually need curl or wget :)

Reply:
Yes but from what I remember ADD cannot be cached as a layer by docker so it's quite a pain when you are rebuilding your image a lot. Perhaps it is not the case anymore.

Reply:
I think I've found a better way (no more `copy` nor `ADD`).